README.md

```diff
@@ -1 +1,108 @@
-# Laravel-Template
+<p align="center">
+  <a href="https://www.aichner-christian.com/" target="_blank" rel="noopener noreferrer">
+    <img src="https://www.aichner-christian.com/img/logo/logo_web.png" alt="Agency Logo" height="150">
+  </a>
+</p>
+
+<h3 align="center">Official Laravel Template</h3>
+
+<p align="center">
+  This repository serves as an entry point for future Laravel projects of the
+  Advertisement Agency Christian Aichner.
+  <br>
+  <br>
+  <a href="https://github.com/aichner/Laravel-Template/issues/new?template=bug_report.md">Report bug</a>
+  ·
+  <a href="https://github.com/aichner/Laravel-Template/issues/new?template=feature_request.md">Request feature</a>
+  ·
+  <a href="https://www.aichner-christian.com/damn/dev">Blog</a>
+  <br>
+  <br>
+  <a href="https://www.codacy.com/app/aichner/Laravel-Template">
+    <img src="https://api.codacy.com/project/badge/Grade/579c145ee6cf4d7e8ae7c1c78a13617a" />
+  </a>
+</p>
+
+<a href="https://www.buymeacoffee.com/M4SVRWQ" target="_blank"><img src="https://bmc-cdn.nyc3.digitaloceanspaces.com/BMC-button-images/custom_images/yellow_img.png" alt="Buy Me A Coffee" style="height: auto !important;width: auto !important;" ></a>
+
+
+## Table of contents
+
+- [Quick start](#quick-start)
+- [Bugs and feature requests](#bugs-and-feature-requests)
+- [Contributing](#contributing)
+- [Community](#community)
+- [Versioning](#versioning)
+- [Creators](#creators)
+- [Thanks](#thanks)
+- [Copyright and license](#copyright-and-license)
+
+## [](#quick-start)Quick start
+
+- Install composer (if not installed yet)
+- Clone the repository: `git clone https://github.com/aichner/Laravel-Template.git Laravel-Template`
+- Go to the new folder `cd Laravel-Template`
+- Run `composer update`
+- Copy `.env.template` and rename it to `.env`
+- Configure your `.env` to your needs
+- Generate key `php artisan key:generate`
+
+The demo site will now be accessible at your local webservice.
+
```

Comment:
We should say too, that the public folder has to be the document root of the webserver.